opennms-container/minion/container-fs/confd/templates/org.opennms.minion.process-env.tmpl

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
We should surround this whole block with a conditional to check if any keys exist under `/process-env/java-opts/` so that we don't output any content if there isn't any.

That way the `remove-if-empty` will work as intended.